Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -382,7 +382,7 @@ open class _ChatMessageContentView<ExtraData: ExtraDataTypes>: _View, ThemeProvi
         }
 
         // Metadata
-        onlyVisibleForYouContainer?.isVisible = content?.onlyVisibleForCurrentUser ?? false
+        onlyVisibleForYouContainer?.isVisible = content?.isOnlyVisibleForCurrentUser ?? false
```

Comment:
```suggestion
        onlyVisibleForYouContainer?.isVisible = content?.isOnlyVisibleForCurrentUser == true
```
I know it is not part of this PR, but let's try to remove the `?? false` whenever we can :)

Reply:
> whenever we can

you mean for all occurrences of `ChatMessage` fields or ...? 🤔

Reply:
✅ Replaced the `??` with `==` for `ChatMessage` fields